mdata/cache/accnt/accnt_test.go

```diff
@@ -0,0 +1,128 @@
+package accnt
+
+import (
+	"testing"
+	"time"
+)
+
+func getNew(maxSize int) *FlatAccnt {
+	a := &FlatAccnt{
+		total:       0,
+		metrics:     make(map[string]*FlatAccntMet),
+		maxSize:     uint64(maxSize),
+		lru:         NewLRU(),
+		evictQ:      make(chan *EvictTarget, 10),
```

Comment:
I think i should probably make `NewFlatAccount()` return an instance of `FlatAccnt` and then add a new method `NewAccount()` which returns the generic interface.